server/datastore/mysql/statistics.go

```diff
@@ -0,0 +1,65 @@
+package mysql
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/kolide/kit/version"
+)
+
+type statistics struct {
+	fleet.UpdateCreateTimestamps
+	Identifier string `db:"anonymous_identifier"`
+}
+
+const (
+	StatisticsFrequency = time.Hour * 24 * 7
```

Comment:
What do you think of moving this to `package fleet`? I don't think this interval would necessarily be mysql specific.